sim/druid/item_sets_pve_phase_6.go

```diff
@@ -201,17 +219,10 @@ func (druid *Druid) applyTAQGuardian4PBonus() {
 	if !druid.HasRune(proto.DruidRune_RuneHandsMangle) {
 		return
 	}
-
-	label := "S03 - Item - TAQ - Druid - Guardian 4P Bonus"
-	if druid.HasAura(label) {
-		return
-	}
-
-	druid.RegisterAura(core.Aura{
-		Label: label,
-		OnInit: func(aura *core.Aura, sim *core.Simulation) {
-			// TODO
-		},
+	druid.OnSpellRegistered(func(spell *core.Spell) {
+		if spell.SpellCode == SpellCode_DruidMangleBear {
+			spell.CD.Duration -= 1500 * time.Millisecond
```

Comment:
```suggestion
			spell.CD.FlatModifier-= 1500 * time.Millisecond
```

Reply:
Oh and also this should go inside of the aura's `OnInit` function to work properly with the set bonus / shoulder rune